react/w-books/src/app/components/FormRow/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Input, { InputProps } from '../Input';
+
+import styles from './styles.module.scss';
+
+interface FormRowProps {
+  className?: string;
+  labelName: string;
+  inputProps: InputProps;
+}
+
+function FormRow({ labelName, className, inputProps }: FormRowProps): JSX.Element {
+  return (
+    <div className={`${styles['form-row']} ${className}`}>
```

Comment:
with the css modules, you can name class with kebab-case in the scss file, but call the class using calmelCase in the HTML tags. 
```suggestion
    <div className={`${styles.formRow} ${className}`}>
```
and same, validate className prop with de clsx